examples/panorama/variables.tf

```diff
@@ -14,7 +14,11 @@ variable "storage_account_name" {
   Default name of the storage account to create.
   The name you choose must be unique across Azure. The name also must be between 3 and 24 characters in length, and may include only numbers and lowercase letters.
   EOF
-  default     = "pantfstorage"
+  type        = string
+}
+
+variable "storage_share_name" {
+  description = "Name of storage File Share to be created that holds `files` for bootstrapping."
```

Comment:
In this case it's not actually true - ie. that it holds files for bootstrap.

Reply:
👍🏻